File: WLIOTAndroidApplication/libwliotproxy/src/main/java/com/alterozoom/libwliotproxy/WLIOTServerProtocolDefs.java
package com.alterozoom.libwliotproxy;

public class WLIOTServerProtocolDefs
{
	public final static ByteArray authenticateSrvMsg=ByteArray.fromString("authenticate");
	public final static ByteArray srvCmdDataMsg=ByteArray.fromString("cmdata");
	public final static ByteArray notifyDeviceIdentifiedMsg=ByteArray.fromString("device_identified");
	public final static ByteArray notifyDeviceLostMsg=ByteArray.fromString("device_lost");
	public final static ByteArray notifyStorageCreatedMsg=ByteArray.fromString("storage_created");
	public final static ByteArray notifyStorageRemovedMsg=ByteArray.fromString("storage_removed");
	public final static ByteArray notifyReloadDevsAndStorsMsg=ByteArray.fromString("reload_devices_and_storages");
	public final static ByteArray vdevMsg=ByteArray.fromString("vdev");
	public final static int controlSslPort=4083;
}

File: wliotproxy-src/libwliotproxy-base/src/WLIOTServerProtocolDefs.cpp
/*******************************************
Copyright 2017 OOO "LMS"

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

	http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.*/

#include "wliot/WLIOTServerProtocolDefs.h"

using namespace WLIOT;

const QByteArray WLIOTServerProtocolDefs::srvCmdDataMsg="cmdata";
const QByteArray WLIOTServerProtocolDefs::authenticateSrvMsg="authenticate";
const QByteArray WLIOTServerProtocolDefs::notifyDeviceIdentifiedMsg="device_identified";
const QByteArray WLIOTServerProtocolDefs::notifyDeviceLostMsg="device_lost";
const QByteArray WLIOTServerProtocolDefs::notifyStorageCreatedMsg="storage_created";
const QByteArray WLIOTServerProtocolDefs::notifyStorageRemovedMsg="storage_removed";
const QByteArray WLIOTServerProtocolDefs::notifyReloadDevsAndStorsMsg="reload_devices_and_storages";
const QByteArray WLIOTServerProtocolDefs::vdevMsg="vdev";
const quint16 WLIOTServerProtocolDefs::controlSslPort=4083;
